test/conformance/cmd_args_test.go

```diff
@@ -0,0 +1,74 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conformance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"github.com/knative/serving/test"
+)
+
+func TestCmdArgsService(t *testing.T) {
+	t.Parallel()
+	clients := setup(t)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   pizzaPlanet1,
+	}
+
+	// Clean up on test failure or interrupt
+	defer test.TearDown(clients, names)
+	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
+
+	text := "THIS IS THE CMD AND ARGS TEST"
+
+	// Setup initial Service
+	_, err := test.CreateRunLatestServiceReady(t, clients, &names, &test.Options{},
+		func(svc *v1alpha1.Service) {
+			c := &svc.Spec.Template.Spec.Containers[0]
+			c.Image = "python"
+			c.Command = []string{"python"}
+			c.Args = []string{"-c", fmt.Sprintf(`
```

Comment:
this is deep 🐺 .

Reply:
Well, `args` better damn well work, right? :)